plasmapy/utils/checks.py

```diff
@@ -409,13 +409,15 @@ def _check_relativistic(V, funcname, betafrac=0.05):
     if not isinstance(V, u.Quantity):
         raise TypeError(errmsg)
 
-    if V.si.unit != u.m / u.s:
+    try:
+        V_over_c = (V / c).to_value(u.dimensionless_unscaled)
+    except Exception:
```

Comment:
Should this not be `u.UnitsError`?

Reply:
Definitely. I'll get a follow-up fix here.

Reply:
Whoops, missed that... Thanks @Cadair